pkg/config/containers.conf

```diff
@@ -886,7 +886,14 @@ default_sysctls = [
 
 # Virtualization provider used to run Podman machine.
 # If it is empty or commented out, the default provider will be used.
-#
+# Linux: QEMU (Default)
+# Windows: there are currently two options:
+#    WSL2    - Windows Subsystem for Linux (Default) 
+#    HyperV  - Windows Server Virtualization
+# Mac: there are currently two options:
+#    vfkit   - Default Apple Hypervisor (Default)
+#    krunkit - Launch virtual machines using the libkrun platform, optimized
+#              for sharing GPU with the machine.
```

Comment:
there should be an empty comment line here to be consistent with the other descriptions above